docs/_includes/fixture-wizard.html

```diff
@@ -0,0 +1,419 @@
+          font-family="HelveticaNeue, Helvetica Neue"
+          font-size="16"
+          font-weight="normal"
+          fill="#000000"
+        >
+          <tspan x="14.244" y="23.5">YES</tspan>
+        </text>
+      </g>
+      <g id="Group" transform="translate(506.000000, 272.000000)">
+        <rect
+          id="Rectangle"
+          fill="#FFFFFF"
+          x="0"
+          y="0"
+          width="58"
+          height="36"
+        ></rect>
+        <text
+          id="NO"
+          fill-rule="nonzero"
+          font-family="HelveticaNeue, Helvetica Neue"
+          font-size="16"
+          font-weight="normal"
+          fill="#000000"
+        >
+          <tspan x="17.144" y="24">NO</tspan>
+        </text>
+      </g>
+      <g id="Group" transform="translate(252.000000, 485.000000)">
+        <rect
+          id="Rectangle"
+          fill="#FFFFFF"
+          x="0"
+          y="0"
+          width="58"
+          height="36"
+        ></rect>
+        <text
+          id="NO"
+          fill-rule="nonzero"
+          font-family="HelveticaNeue, Helvetica Neue"
+          font-size="16"
+          font-weight="normal"
+          fill="#000000"
+        >
+          <tspan x="17.144" y="24">NO</tspan>
+        </text>
+      </g>
+      <polygon
+        id="Path"
+        stroke="#444444"
+        stroke-width="2"
+        fill="#444444"
+        fill-rule="nonzero"
+        points="534.93 577.96 534.6 569.03 537.41 572.5 541.88 572.34"
+      ></polygon>
+      <line
+        x1="700"
+        y1="435"
+        x2="756.85"
+        y2="572.39"
+        id="Path"
+        stroke="#444444"
+        stroke-width="2"
+      ></line>
+      <polygon
+        id="Path"
+        stroke="#444444"
+        stroke-width="2"
+        fill="#444444"
+        fill-rule="nonzero"
+        points="759.15 577.93 752.39 572.07 756.85 572.39 759.78 569.01"
+      ></polygon>
+      <g id="Group" transform="translate(540.000000, 485.000000)">
+        <rect
+          id="Rectangle"
+          fill="#FFFFFF"
+          x="0"
+          y="0"
+          width="58"
+          height="36"
+        ></rect>
+        <text
+          id="YES"
+          fill-rule="nonzero"
+          font-family="HelveticaNeue, Helvetica Neue"
+          font-size="16"
+          font-weight="normal"
+          fill="#000000"
+        >
+          <tspan x="14.244" y="23.5">YES</tspan>
+        </text>
+      </g>
+      <g id="Group" transform="translate(699.000000, 489.000000)">
+        <rect
+          id="Rectangle"
+          fill="#FFFFFF"
+          x="0"
+          y="0"
+          width="58"
+          height="36"
+        ></rect>
+        <text
+          id="NO"
+          fill-rule="nonzero"
+          font-family="HelveticaNeue, Helvetica Neue"
+          font-size="16"
+          font-weight="normal"
+          fill="#000000"
+        >
+          <tspan x="17.144" y="24">NO</tspan>
+        </text>
+      </g>
+    </g>
+  </g>
+</svg>
```

Comment:
It looks good. 
What tools do you use for making this diagram? Writing a guide as a comment for how we can modify this diagram is better for later.

Reply:
I used Sketch for this.  I can add the source `.sketch` file to the repo